### PR TITLE
Added change to send notification based on job status

### PR DESCRIPTION
**Pull request by tisutisu**
This PR will address story https://issues.redhat.com/browse/HELM-444 
Also added change to stop CodeQL analysis job to run on forked repos.

### CODE REVIEW

**Review by mmulholla — CHANGES_REQUESTED** [review]
A couple of minor comments.

File: .github/workflows/version_check.yml [hunk 1]
```diff
@@ -405,19 +419,35 @@ jobs:
...
-      - name: Send message to slack channel
-        id: notify
-        if: always() && github.event_name == 'schedule' && steps.check_test.outputs.run_tests == 'true'
+      - name: Send message to helm_dev slack channel
+        id: notify_dev
+        if: ${{ github.event_name == 'schedule' && steps.check_test.outputs.run_tests == 'true' && steps.run-schedule-tests.conclusion != 'success' }}
         uses: archive/github-actions-slack@v2.0.0
         env:
           SOFTWARE_NAME: "chart-verifier"
           SOFTWARE_VERSION: ${{ steps.get_curr_cv_version.outputs.current_cv_digest }}
```
[the remark this answers is not in the document]
Comment (tisutisu) [reply .github/workflows/version_check.yml:428]: Removed the env variables as they are not needed in the latest commit

File: .github/workflows/version_check.yml [hunk 2]
```diff
@@ -405,19 +419,35 @@ jobs:
...
+        if: ${{ github.event_name == 'schedule' && steps.check_test.outputs.run_tests == 'true' && steps.run-schedule-tests.conclusion != 'success' }}
+        run: echo "The result was ${{ steps.notify_dev.outputs.slack-result }}"
+      
+      - name: Send message to helm_notify slack channel
+        id: notify
+        if: ${{ github.event_name == 'schedule' && steps.check_test.outputs.run_tests == 'true' && steps.run-schedule-tests.conclusion == 'success' }}
+        uses: archive/github-actions-slack@v2.0.0
+        env:
+          SOFTWARE_NAME: "chart-verifier"
+          SOFTWARE_VERSION: ${{ steps.get_curr_cv_version.outputs.current_cv_digest }}
```
[the remark this answers is not in the document]
Comment (tisutisu) [reply .github/workflows/version_check.yml:444]: Removed the env variables as they are not needed in the latest commit

**Review by mmulholla — APPROVED** [review]
LGTM

**Pull request merged**